gizmos/tree.py

```diff
@@ -1101,10 +1081,10 @@ def renderOwlClassExpression(given_rows, rel=None):
 
 
 def row2po(stanza, data, row, db):
-    """Convert a predicate and object from a sqlite query result row to hiccup-style HTML."""
+    """Convert a predicate and object/value from a sqlite query result row to hiccup-style HTML."""
     predicate = row["predicate"]
     predicate_label = data["labels"].get(predicate, predicate)
-    p = ["a", {"href": curie2href(predicate, db)}, predicate_label]
+    p = ["b", ["a", {"href": curie2href(predicate, db)}, predicate_label]]
```

Comment:
I don't want the predicates to be bold.

Reply:
They blend in with the object, if the object is a CURIE or IRI. Is that OK?

Reply:
If it's a nested list, then it won't blend in.